front/app/modules/commercial/content_builder/admin/containers/index.tsx

```diff
@@ -14,49 +14,57 @@ import ContentBuilderToolbox from '../components/ContentBuilderToolbox';
 import ContentBuilderSettings from '../components/ContentBuilderSettings';
 import Container from '../components/CraftComponents/Container';
 import RenderNode from '../components/RenderNode';
+import ContentBuilderTopBar from '../components/ContentBuilderTopBar';
 
 const Wrapper = styled.div`
   flex: 0 0 auto;
-  width: 210px;
+  width: 212px;
 `;
 
 const ContainerInner = styled.nav`
   flex: 0 0 auto;
-  width: 210px;
+  width: 212px;
   display: flex;
   flex-direction: column;
   align-items: center;
   position: fixed;
```

Comment:
Seems we are still relying in the position fixed here for the layout?

Reply:
@IvaKop Correct - this used the position prop prior to me adding the top bar component. Should it be changed?